Deployment/Release practices.md

```diff
@@ -41,6 +44,11 @@ If the app is yet to see its first release to production, the store listing will
 
 By using open tracks, user feedback can be gathered directly from the Play Store, but are not public, meaning that they won't affect your total production listing.
 
+### Pre-registration track
+
+With this track you enable users to pre register for your app before it is actually available. When it becomes available it will be automatically installed to all the pre-registered users.
+It is mostly used to boost pre-launch game campaigns and improve retention metrics.
```

Comment:
```suggestion
With this track, you enable users to pre-register for your app before it is actually available. When the app becomes available, it will be automatically installed to all the pre-registered users.
It is mostly used to boost pre-launch game campaigns and improve retention metrics.
```